lib/src/sturdy_http.dart

```diff
@@ -240,22 +241,6 @@ Dio _configureDio({
         return dio..httpClientAdapter = customAdapter;
       }
       return dio;
-    })
-    ..map((dio) {
```

Comment:
This type of configuration means proxy changes at runtime weren't reflected. Instead, consumers can provide this via an `Interceptor`.